test/verify/check-machines-dbus

```diff
@@ -58,6 +58,44 @@ class TestMachinesDBus(machineslib.TestMachines):
         if self.machine.image == "ubuntu-stable":
             self.machine.execute("chmod o+rwx /run/libvirt/libvirt-sock")
 
+    def testAutostart(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-autostart-checkbox")
+
+        # set checkbox state and check state of checkbox
+        b.set_checked("#vm-subVmTest1-autostart-checkbox", True) # don't know the initial state of checkbox, so set it to checked
+        b.wait_present("#vm-subVmTest1-autostart-checkbox:checked")
+        # check virsh state
+        autostartState = m.execute("virsh dominfo subVmTest1 | sed -n '/^Autostart/p' | sed -e 's/ +//' -e 's/Autostart://'").strip()
+        self.assertEqual(autostartState, "enable" )
+
+        # change checkbox state and check state of checkbox
+        b.click("#vm-subVmTest1-autostart-checkbox")
+        b.wait_not_present("#vm-subVmTest1-autostart-checkbox:checked")
+        # check virsh state
+        autostartState = m.execute("virsh dominfo subVmTest1 | sed -n '/^Autostart/p' | sed -e 's/ +//' -e 's/Autostart://'").strip()
+        self.assertEqual(autostartState, "disable")
+
+        # change checkbox state and check state of checkbox
+        b.click("#vm-subVmTest1-autostart-checkbox")
+        b.wait_present("#vm-subVmTest1-autostart-checkbox:checked")
+        # check virsh state
+        autostartState = m.execute("virsh dominfo subVmTest1 | sed -n '/^Autostart/p' | sed -e 's/ +//' -e 's/Autostart://'").strip()
```

Comment:
What about "virsh dominfo subVmTest1 | grep 'Autostart:' | awk '{print $2}''
I think it's cleaner than the double sed. And you don't need strip.
This is my last comment, otherwise great!